### PR TITLE
Modify zoom behaviour using FitVertical mode

### PR DESCRIPTION
**Pull request by EvilSquirrel**
Related to issue #14 (Use 'fit-to-width' mode when in landscape).

As mentioned this is my first crack at using Git and Github, so let me know if I messed up somehow.

### CODE REVIEW

File: Classes/ComicViewController.m [hunk 1]
```diff
@@ -30,16 +30,21 @@
...
 
+typedef enum { kPositionCenter, kPositionLeftEdge, kPositionRightEdge } ContentPos;
+
 @interface ComicDocumentView : DocumentView
 @end
 
 @interface ComicPageView : ZoomView {
 @private
   NSString* _file;
+  bool _contentIsLandscape;
```
Comment (swisspol) [inline Classes/ComicViewController.m]: ObjC uses `BOOL`

**Comment by swisspol** [comment]
It's a big change overall and I'm really concerned about introducing regressions (ComicFlow has a big user base). On top of that, dealing with UIViews orientation and positioning can be quite tricky and it still has to work all the way to iOS 5 which ComicFlow still supports.

File: Classes/ComicViewController.m (hunk 1 above)
Comment (EvilSquirrel) [inline Classes/ComicViewController.m]: I know, that's my C++ leaking through :)

**Comment by EvilSquirrel** [comment]
That's understandable, and part of why I never shared my changes before - I wasn't concerned with having it work for everyone, just me... That being said, I don't think there's anything too funky in there, so it's more an issue of "compatibility with old bugs" :)

**Comment by swisspol** [comment]
I'm sorry but this PR is too big for me to actually review it in depth and assess the risks. It's too risky to merge this in. Thanks for understanding.

**Pull request closed without merging**